jlm/llvm/Makefile.sub

```diff
@@ -5,6 +5,7 @@ libllvm_SOURCES = \
     jlm/llvm/backend/jlm2llvm/instruction.cpp \
     jlm/llvm/backend/jlm2llvm/jlm2llvm.cpp \
     jlm/llvm/backend/jlm2llvm/type.cpp \
+    jlm/llvm/backend/rvsdg2dot/rvsdg2dot.cpp \
```

Comment:
Let's just call it `dot`: `jlm/llvm/backend/dot`. I know the other ones are named differently right now, but I want to move away from this.